### PR TITLE
No JIRA.  Update rancher chart to v2.5.9

### PR DESCRIPTION
**Pull request by michael-cico**
# Description

Update the Rancher chart to v2.5.9
- Fix rancher cluster role cleanup

# Checklist 

As the author of this PR, I have:

- [x] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

**Review by pfmackin — APPROVED** [review]

File: platform-operator/thirdparty/charts/rancher/values.yaml [hunk 1]
```diff
@@ -136,3 +136,18 @@ customLogos:
   # storageClass: "-"
   accessMode: ReadWriteOnce
   size: 1Gi
+
+# Rancher post-delete hook
+postDelete:
+  enabled: true
+  image:
+    repository: rancher/shell
```
Comment (fredtibbitts) [inline platform-operator/thirdparty/charts/rancher/values.yaml:144]: Does this require an entry in the BOM? I don't know the answer.

**Review by fredtibbitts — COMMENTED** [review]
There are a few empty files, which seems odd. Are those supposed to be deleted?

File: platform-operator/thirdparty/charts/rancher/values.yaml (hunk 1 above)
Comment (michael-cico) [reply platform-operator/thirdparty/charts/rancher/values.yaml:144]: I'm not sure...the private registry tests worked, but that may not mean much.  I'm re-running those, I'll have to look to see if it's using the private registry versions or not.

Comment (michael-cico) [reply platform-operator/thirdparty/charts/rancher/values.yaml:144]: I think we're covered, I see this image/tag getting uploaded during the test run

```
Processing image docker.io/rancher/shell:v0.1.6 to phx.ocir.io/****/verrazzano-private-registry/mcico/update-rancher-chart-v259/b6/rancher/shell:v0.1.6
>> Tagging image: docker.io/rancher/shell:v0.1.6 to phx.ocir.io/****/verrazzano-private-registry/mcico/update-rancher-chart-v259/b6/rancher/shell:v0.1.6
```

**Review by pfmackin — APPROVED** [review]

**Review by fredtibbitts — APPROVED** [review]

**Pull request merged**